.github/workflows/workflow.yml

```diff
@@ -216,21 +216,56 @@ jobs:
       - name: Prepare build artifacts [Windows]
         if: matrix.os == 'windows-latest'
         run: |
-          cd target/${{ matrix.target }}/release
-          strip starship.exe
-          7z a ../../../${{ matrix.name }} starship.exe
-          cd -
+          strip target/${{ matrix.target }}/release/starship.exe
+          7z a ${{ matrix.name }} target/${{ matrix.target }}/release/starship.exe
 
       - name: Prepare build artifacts [-nix]
         if: matrix.os != 'windows-latest'
         run: |
-          cd target/${{ matrix.target }}/release
-          strip starship
-          tar czvf ../../../${{ matrix.name }} starship
-          cd -
+          strip target/${{ matrix.target }}/release/starship
+          tar czvf ${{ matrix.name }} target/${{ matrix.target }}/release/starship
+
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: ${{ matrix.name }}
+          path: ${{ matrix.name }}
+
+  # Create GitHub release with Rust build targets and release notes
+  github_release:
+    if: startsWith(github.ref, 'refs/tags/v')
+    name: Create GitHub Release
+    needs: github_build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Download releases from github_build
+        uses: actions/download-artifact@v1
+        with:
+          name: starship-x86_64-unknown-linux-gnu.tar.gz
+          path: .
+      - name: Download releases from github_build
+        uses: actions/download-artifact@v1
+        with:
+          name: starship-x86_64-unknown-linux-musl.tar.gz
+          path: .
+      - name: Download releases from github_build
+        uses: actions/download-artifact@v1
+        with:
+          name: starship-x86_64-apple-darwin.tar.gz
+          path: .
+      - name: Download releases from github_build
+        uses: actions/download-artifact@v1
+        with:
+          name: starship-x86_64-pc-windows-msvc.zip
+          path: .
+
+      - shell: bash
+        run: |
+          openssl dgst -sha256 -r starship-* | sed 's/\*//' >> CHECKSUMS.sha256
```

Comment:
Should we perhaps use `$filename.sha256`?
Is a single `.sha256` file used by applications?

Reply:
That's how I had it working at the beginning, but looking around other projects seem to lump them all together (i.e. [veracode](https://launchpadlibrarian.net/448779175/veracrypt-1.24-Hotfix1-sha256sum.txt)).

The filename and format were chosen from the powersell line at the end of this comment https://github.com/starship/starship/pull/649#issuecomment-554659018 

It would be trivial to split them up though and I'm more than happy to change it to whichever is easiest/best, so either way is just as well for me. @r15ch13 does scoop have a preference, or rather, which option would make life easier for scoop?

Reply:
Providing single checksum files or one contain all is up to you. Scoop can handle both cases with ease. :smile: